operator/config/openshift/patch_manager_env.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - env:
+        - name: MANAGER_CREATE_RESOURCES
+          value: "true"
+        - name: ENGINE_CONTAINER_USER
+          value: ""          
```

Comment:
Would this not need to be 8888 by default?

Reply:
For openshift it needs not to be set as Openshift sets the user